vispy/ipython.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""Entry point for vispy's IPython bindings"""
+
+import IPython
```

Comment:
@Eric89GXL but, the two pieces of code aren't exactly the same, right?

The test case decorator causes the test case to skip. Here, we need to throw a human-readable error for the user. I'm not sure what you want me to reuse